lms/static/sass/discussion/_discussion.scss

```diff
@@ -324,27 +341,17 @@ body.discussion {
 }
 
 .discussion-show {
+  @include float(right);
   position: relative;
   top: 3px;
   font-size: $forum-base-font-size;
   text-align: center;
 
-  &.shown {
-    .show-hide-discussion-icon {
-      background-position: 0 0;
-    }
+  &.btn.btn-brand.shown {
```

Comment:
I notice this button's font family is sans-serif, instead of OpenSans in production.
Currently in LMS, only .btn has OpenSans, so I added .btn. But you're right on btn-brand is not needed.